Sources/AsyncHTTPClient/ConnectionPool/State Machine/HTTPConnectionPool+HTTP2StateMachine.swift

```diff
@@ -33,16 +34,19 @@ extension HTTPConnectionPool {
         private let idGenerator: Connection.ID.Generator
 
         private(set) var lifecycleState: StateMachine.LifecycleState
+        private let retryConnectionEstablishment: Bool
```

Comment:
Can we add a comment here, why we have this property. We should note wherever we can, that this is a dull setting. :P